pypowsybl/network/impl/perunit.py

```diff
@@ -519,6 +528,20 @@ def update_lcc_converter_station(self, df: pd.DataFrame = None, **kwargs: ArrayL
         self._un_per_unit_i(to_update, ['i'], nominal_v)
         self._network.update_lcc_converter_stations(to_update)
 
+    def update_ratio_tap_changers(self, df: pd.DataFrame = None, **kwargs: ArrayLike) -> None:
+        to_update = _adapt_to_dataframe(ElementType.RATIO_TAP_CHANGER, df, **kwargs).copy()
+        ref = self._network.get_ratio_tap_changers(all_attributes=True)
+        voltage_levels = ref[[]].merge(
+            self.get_2_windings_transformers()[['voltage_level1_id', 'voltage_level2_id']],
+            left_index=True, right_index=True)
+        nominal_v1 = self._get_indexed_nominal_v(voltage_levels, 'voltage_level1_id')
+        nominal_v2 = self._get_indexed_nominal_v(voltage_levels, 'voltage_level2_id')
+        regulated_side_nominal_voltage = nominal_v2
```

Comment:
Why do we get a different peruniting depending on if we have the target_v inside the dataframe?